modules/core/src/scenegraph/gltf/gltf-instantiator.js

```diff
@@ -155,15 +155,22 @@ export default class GLTFInstantiator {
     return loadedAttributes;
   }
 
-  createBuffer(bufferView, target) {
+  createBuffer(attribute, target) {
+    if (!attribute.bufferView) {
+      // Draco decoded files do not have a bufferView
+      attribute.bufferView = {};
+    }
+
+    const {bufferView} = attribute;
     if (!bufferView.lumaBuffers) {
       bufferView.lumaBuffers = {};
     }
 
     if (!bufferView.lumaBuffers[target]) {
       bufferView.lumaBuffers[target] = new Buffer(this.gl, {
         id: `from-${bufferView.id}`,
-        data: bufferView.data,
+        // Draco decoded files have attribute.value
+        data: bufferView.data || attribute.value,
```

Comment:
Yep, seems like this was returned by DracoDecoder, should remap this to fit with glTF model.

Reply:
Yes please.